.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All of these .idea files are not necessary. They help render the project in your local IDE. Please remove any .idea/ files. We should add this to .gitignore to avoid in future pushes.